javascript/javascript3/week1/weather/index.js

```diff
@@ -0,0 +1,102 @@
+const iconElement = document.querySelector(".weather-icon");
+const tempElement = document.querySelector(".temperature-value p");
+const descElement = document.querySelector(".temperature-description p");
+const locationElement = document.querySelector(".location p");
+const notificationElement = document.querySelector(".notification");
+
+// date
+n = new Date();
+y = n.getFullYear();
+m = n.getMonth() + 1;
+d = n.getDate();
```

Comment:
I think you have forgotten to add `const` at the beginning of these lines to declare new variables